pkg/v1/tkg/client/machine_deployment_test.go

```diff
@@ -1181,6 +1181,25 @@ var _ = Describe("Machine Deployment", func() {
 						},
 					},
 				}
+				md2 = capi.MachineDeployment{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "test-cluster-np-1",
```

Comment:
not sure if it is intentional, but it is weird to set both to the same name, also weird later to modify md2's name to match np1Name